### PR TITLE
docs: review communities REST API docs

### PR DESCRIPTION
**Pull request by effervescent-shot**
Closes https://github.com/inveniosoftware/invenio-communities/issues/316

### CODE REVIEW

File: docs/install/run.md [hunk 1]
```diff
@@ -447,6 +446,308 @@ curl -k -XPOST https://127.0.0.1:5000/api/records/jnmmp-51n47/draft/actions/publ
...
+    "record_policy": "open"
+  },
+  "id": "yet_another_community",
+  "metadata": {
+    "title": "Brand New Community",
+    "description": "This is an example Community.",
+    "type": "event",
+    "curation_policy": "This is the kind of records we accept.",
+    "page": "Information for my community.",
+    "website": "https://inveniosoftware.org/"
```
Comment (slint) [inline docs/install/run.md:695]: You can use a real example here, e.g. "Awesome Repositories Conf 2021", with an appropriate description/curation_policy, etc.

File: docs/install/run.md [hunk 2]
```diff
@@ -447,6 +446,308 @@ curl -k -XPOST https://127.0.0.1:5000/api/records/jnmmp-51n47/draft/actions/publ
...
+  }
+}'
+```
+
+#### Update a community
+
+You can renamed a community using the API.
```
Comment (slint) [inline docs/install/run.md:702]:
```suggestion
You can update a community's profile and access settings using the API. 
```

File: docs/install/run.md [hunk 3]
```diff
@@ -447,6 +446,308 @@ curl -k -XPOST https://127.0.0.1:5000/api/records/jnmmp-51n47/draft/actions/publ
...
+  "id": "yet_another_community",
+  "metadata": {
+    "title": "Updated Comm Name"
+  }
+}'
+```
+
+### Rename a community
+
+You can update the community, access and metadata fields using the API:
```
Comment (slint) [inline docs/install/run.md:720]: I think you swapped the above description with this one :)

File: docs/install/run.md [hunk 4]
```diff
@@ -462,3 +763,6 @@ invenio-cli services stop
 
 Check the [Cleanup section](./cleanup.md) if you wish to remove every
 reference to InvenioRDM from Docker (containers, images, networks, etc.).
+
+
+
```
Comment (slint) [inline docs/install/run.md:768]: One empty line at the bottom is good.
```suggestion

```

File: docs/reference/rest_api.md [hunk 5]
```diff
@@ -2076,54 +2080,61 @@ Content-Type: application/json
 HTTP/1.1 201 CREATED
 Content-Type: application/json
 
-{
+{
```
Comment (slint) [inline docs/reference/rest_api.md:2083]:
```suggestion
{
```

There's some extra whitespace here

File: docs/reference/rest_api.md [hunk 6]
```diff
@@ -2076,54 +2080,61 @@ Content-Type: application/json
...
-{
+{   
   "access": {
-    "visibility": "public",
-    "member_policy": "open",
-    "record_policy": "open",
     "owned_by": [
       {
-          "user": {user-id}
+        "user": <user_id>
```
Comment (slint) [inline docs/reference/rest_api.md:2087]: For consistency, I would keep the curly brackets as in the records examples

File: docs/reference/rest_api.md [hunk 7]
```diff
@@ -2257,95 +2278,225 @@ Content-Type: application/json
...
+        "name": "CERN",
+        "identifiers": [
+          {
+            "identifier": "01ggx4157", 
+            "scheme": "ror"
+          }
+        ]
+      }
+    ]
+  }
```
Comment (slint) [inline docs/reference/rest_api.md:2404]: I think for the rename only the `id` field is needed. The response does have the full payload response though

**Review by slint — COMMENTED** [review]
LGTM, some minor fixes and nits

**Comment by fenekku** [comment]
This hasn't seen activity in a year and we have updated the community API recently. I will close. Feel free to submit another with salvaged content if some is worthwhile.

**Pull request closed without merging**